tests/integration/test_orgs_invites.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+"""Integration tests for invitation methods implemented on orgs."""
+import github3
+
+from .helper import IntegrationHelper
+
+
+class TestInvitations(IntegrationHelper):
+
+    """Team integration tests."""
+
+    betamax_kwargs = {'match_requests_on': ['method', 'uri', 'json-body']}
+
+    def setUp(self):
+        super(TestInvitations, self).setUp()
+        self.basic_login()
+
+    def get_organization(self, name='mozillatw'):
+        """Get the organization for each test."""
+        self.token_login()
+        o = self.gh.organization(name)
+        assert isinstance(o, github3.orgs.Organization)
+        return o
+
+    def test_invites(self):
```

Comment:
Ah - I think I get how the cassettes work now. Certainly enough for me to make another pass. Thanks!

> Can you link to this? I think I phrased it extremely poorly. What I don't want is existing tests to be appended to. I want new tests written.

Here is the text:

> Please do not add your regression test to an existing test, but create a new one.

I was interpreting this as "do not add .. to an existing test *file*", and your intent was to not add to an existing test *case*. So I read it very liberally :/